Classes/Controller/ModuleController.php

```diff
@@ -336,6 +324,16 @@ public function fileReferenceAction($element = 0)
         );
     }
 
+    /**
+     * Returns the global database connection object.
+     *
+     * @return \TYPO3\CMS\Core\Database\DatabaseConnection
+     */
+    protected function getDatabaseConnection()
+    {
+        return $GLOBALS['TYPO3_DB'];
```

Comment:
Uh no, not anymore. ;-)

Reply:
strange, actually I removed the part. stange that it looks as if I added it...

Reply:
it is removed

Reply:
This still is marked as "added" ... :thinking:

Reply:
I remove it in another pull request. I split up the changes into chunks of one topic. the uribuiler doesnt touch the db things